ssl/statem/statem_clnt.c

```diff
@@ -1183,7 +1183,6 @@ int tls_construct_client_hello(SSL *s, WPACKET *pkt)
 
     /* TLS extensions */
     if (!tls_construct_extensions(s, pkt, SSL_EXT_CLIENT_HELLO, NULL, 0, &al)) {
-        ssl3_send_alert(s, SSL3_AL_FATAL, al);
```

Comment:
This is in the construction of the initial ClientHello - i.e. before we've started any kind of handshake at all. It would be a bit weird for a client to open a connection to a server, just send an alert and then close down again. The alert really signifies the end of a TLS connection - but we haven't started one at this point. I removed the alert here for consistency with the rest of this function. None of the other construction errors in this function result in an alert being sent - so why do that for the extension construction step?